jobs/clean_builds.groovy

```diff
@@ -20,9 +20,9 @@ import util.CleanBuild
     seedJob: SEED_JOB,
   ],
   [
-    name: 'dax/dax_webserv',
-    products: 'dax_webserv',
-    buildConfig: 'dax-lsstsw-matrix',
+    name: 'dax/webserv',
+    products: ['dax_imgserv', 'dax_metaserv'],
```

Comment:
`products` needs to be a space separated string rather than an array.

Reply:
Fixed.